R/utils.R

```diff
@@ -2,6 +2,17 @@ last <- function(x) x[length(x)]
 
 sq <- function(x) glue::single_quote(x)
 
+sglue <- function(...) stop(glue::glue(..., .envir = parent.frame()), call. = FALSE)
```

Comment:
I think it's because, if you wrap `glue()`, you need to preserve and convey the correct environment for the template variables. `.envir` is not a simple, static default value.